.gitmodules

```diff
@@ -24,3 +24,11 @@
 	path = kube-proxy
 	url = https://github.com/openshift/kubernetes
 	branch = sdn-4.9-kubernetes-1.22.0-rc.0
+[submodule "containerd"]
+	path = containerd
+	url = https://github.com/openshift/containerd
+	branch = release/1.5
+[submodule "hcsshim"]
+	path = hcsshim
+	url = https://github.com/openshift/hcsshim
+	branch = master
```

Comment:
are we not suppose to point to specific version instead of master ?

Reply:
+1
Commit message mentions `v0.9.0-rc0`

Reply:
> are we not suppose to point to specific version instead of master ?

I don't think so, there is no such branch as `v0.9.0-rc0` .

[v0.9.0-rc0](https://github.com/openshift/hcsshim/releases/tag/v0.9.0-rc0) is a tag.

Reply:
hcsshim does not seem to follow branching. As @jrvaldes pointed out, v0.9.0-rc0 is a tag which I checked out.

Reply:
Actually this is local RedHat Repo. we can ask the hcsshim owners to have some branching strategy.  I think we should not wait for it. lets go ahead and merge it.

Reply:
PS: anything we use in OpenShift is forked into the OpenShift github organization but we try not to add any carry patches. We never reference or use an upstream repo directly. Sure we can ask the hcsshim owners for a branching strategy 🤞🏽